src/meta/KVBasedClusterIdMan.h

```diff
@@ -51,7 +51,7 @@ class ClusterIdMan {
       ::close(fd);
       return false;
     }
-    LOG(INFO) << "Persiste clusterId " << clusterId << " succeeded!";
+    LOG(INFO) << "Persist clusterId " << clusterId << " succeeded!";
```

Comment:
Here I picked `Persist` instead of `Persisted` -- I should have checked the whole set.